src/test/java/org/alfresco/test/APIExplorerIntegrationTest.java

```diff
@@ -141,6 +144,20 @@ public void testSearchAPIDefinition() throws Exception {
         validateSearchDefn(definitionUrl + YAML);
         validateSearchDefn(definitionUrl + JSON);
     }
+    @Test

```

Comment:
There is too much copy paste code in this test class. We could have extracted a lot into a private method and passed just the `definitionUrl `.  But it's not something you introduce. It could be an improvement we could make.

Reply:
Yes, I agree it would have been nice to refactor it.